src/test/java/seedu/duke/commands/BalanceCommandTest.java

```diff
@@ -90,8 +90,8 @@ public void getBalance_ifCurrencyIsNotSpecified_shouldReturnAllCurrencies() {
                     account
             );
             assertEquals(2, output.size());
-            assertEquals(200, 100 * output.get(Currency.CNY).getBalance());
-            assertEquals(40, 100 * output.get(Currency.EUR).getBalance());
+            assertEquals(200,  output.get(Currency.CNY).getBalance()/100);
+            assertEquals(40, output.get(Currency.EUR).getBalance()/100);
```

Comment:
I have doubts about this implementation because this makes it seemed that we are trying to put in magic literals to make the unit tests and pass, telling the future developers that the output is already incorrect. Try see if you can make the tests pass without adding a `/100`. As mentioned previously, one fix would be to keep the `/ 100.0f ` and multiply the balance by 100 in the constructor when assigning it. I have attempted this fix in another branch.